Resources/PinConfigs.kext/Contents/Info.plist

```diff
@@ -6079,7 +6079,7 @@
 					<key>Codec</key>
 					<string>InsanelyDeepak - Conexant CX20756</string>
 					<key>CodecID</key>
-					<integer>351359247</integer>
+					<integer>351359252</integer>
```

Comment:
Is this change intentional? I get a feeling it may not be.

Reply:
well the CodecID in PinConfigs is the wrong one so i corrected it with the right one